src/termio/Exec.zig

```diff
@@ -587,10 +582,18 @@ const Subprocess = struct {
     grid_size: renderer.GridSize,
     screen_size: renderer.ScreenSize,
     pty: ?Pty = null,
-    command: ?Command = null,
-    flatpak_command: ?FlatpakHostCommand = null,
+    process: ?Process = null,
     linux_cgroup: Command.LinuxCgroup = Command.linux_cgroup_default,
```

Comment:
Considering making this part of the fork/exec state.

Reply:
This actually gets set before the process so it has nothing to do with that so not going to do that.